src/examples/elementary/meson.build

```
@@ -121,8 +121,15 @@ examples = [
   'efl_canvas_textblock_obstacles_example'
 ]
 
+examples_deps = [elementary, ecore, eio]
+
+if get_option('x11')
```

maybe add an if not windows?
```suggestion
if get_option('x11') and not sys_windws
```

---

Is it possible to get option **x11** when **windows**?